UI/window-basic-main.cpp

```diff
@@ -82,6 +82,7 @@
 
 using namespace json11;
 using namespace std;
+#include <set>;
```

Comment:
Probably best to put this alongside `fstream` and `sstream`, rather than below `using namespace`.

Reply:
OK, I did. Also, there were duplicate imports of `fstream` and `sstream` -- I removed them.